amundsen_application/api/metadata/v0.py

```diff
@@ -634,3 +634,70 @@ def _get_dashboard_metadata(*, uri: str, index: int, source: str) -> None:
         message = 'Encountered exception: ' + str(e)
         logging.exception(message)
         return make_response(jsonify({'dashboard': {}, 'msg': message}), HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+@metadata_blueprint.route('/table/<table_key>/dashboards', methods=['GET'])
+def get_related_dashboard_metadata(table_key) -> Response:
+    """
+    Call metadata service endpoint to fetch related dashboard metadata
+    :return:
+    """
+    try:
+        # import ipdb; ipdb.set_trace()
+        # table_key = get_query_param(request.args, 'key')
+        import logging
+        logging.basicConfig(level=logging.DEBUG)
+        logging.info(table_key)
+
+        url = f'{app.config["METADATASERVICE_BASE"]}{TABLE_ENDPOINT}/{table_key}/dashboard/'
+        results_dict = _get_related_dashboards_metadata(table_key=table_key, url=url)
```

Comment:
```suggestion
        results_dict = _get_related_dashboards_metadata(url=url)
```
I see `table_key` currently isn't used in the method.

Reply:
Good point, it comes from a previous version where I was building the url within that method.
Thanks!